lcm_status_channel: "IIWA_STATUS"
lcm_command_channel: "IIWA_COMMAND"
lcm_plan_channel: "COMMITTED_ROBOT_PLAN"
lcm_stop_channel: "STOP"
num_joints: 7
robot_ee_body_name: "iiwa_link_ee"
robot_urdf_path: "${SPARTAN_SOURCE_DIR}/drake/manipulation/models/iiwa_description/urdf/iiwa14_no_collision.urdf"
joint_speed_limit_degree_per_sec: 300.0 # This is trivially large, but most plans generated by IK aren't commanding such speed.
control_period_s: 0.005

task_space_plan:
  kp_rotation: [50, 50, 50] # orientation P gains
  kp_translation: [100, 100, 100] # translation P gains

joint_limit_tolerance: 5.0 # subtract this from measured joint limits before sending command
joint_limits:
  iiwa_joint_1: [-170, 170]
  iiwa_joint_2: [-120, 120]
  iiwa_joint_3: [-170, 170]
  iiwa_joint_4: [-120, 120]
  iiwa_joint_5: [-170, 170]
  iiwa_joint_6: [-120, 120]
  iiwa_joint_7: [-175, 175]